### PR TITLE
docs: Extend proposal 3 with git interim solution

### PR DESCRIPTION
**Pull request by rossf7**
#### What type of PR is this?

kind/documentation

#### What this PR does / why we need it:

Extends proposal 3 with git interim solution and example JSON file.

### CODE REVIEW

**Review by nikimanoledaki — APPROVED** [review]
Amazing - this is also the first time that we see a schema for the metrics to be stored 😍

**Review by dipankardas011 — APPROVED** [review]
LGTM

File: docs/proposals/proposal-003-report.md [hunk 1]
```diff
@@ -276,31 +275,96 @@ Collect metrics by making curl requests to the `/metrics` endpoints.
...
+                "value": 953597074.2857143
+            },
+            {
+                "name": "container_memory_working_set_bytes",
+                "value": 1070605165.7142856
+            },
+            {
+                "name": "kepler_container_joules_total",
+                "value": 7867.507499998423
+            }
```
Comment (dipankardas011) [inline docs/proposals/proposal-003-report.md:321]: Okay so these are the metrics from the prometheus?

File: docs/proposals/proposal-003-report.md [hunk 2]
```diff
@@ -276,31 +275,96 @@ Collect metrics by making curl requests to the `/metrics` endpoints.
...
+                    "value": 78.81
+                },
+                "embodied_carbon": {
+                    "units": "gCO2eq",
+                    "value": 3.92
+                },
+                "pue": {
+                    "provider": "Equinix",
+                    "value": 1.42 
+                }
```
Comment (dipankardas011) [inline docs/proposals/proposal-003-report.md:341]: and these are constants we are using at that time for sci calculations?

**Comment by locomundo** [comment]
great!

**Pull request merged**